pkg/controller/migmigration/hooks.go

```diff
@@ -235,6 +235,16 @@ func (t *Task) baseJobTemplate(hook migapi.MigPlanHook, migHook migapi.MigHook)
 						{
 							Name:  strings.ToLower(t.PlanResources.MigPlan.Name + "-" + hook.Phase),
 							Image: migHook.Spec.Image,
+							Env: []corev1.EnvVar{
+								{
+									Name:  "MIGRATION_NAMESPACES",
+									Value: strings.Join(t.PlanResources.MigPlan.Spec.Namespaces, ","),
+								},
+								{
+									Name:  "MIGRATION_NAME",
+									Value: t.PlanResources.MigPlan.Name,
```

Comment:
@jmontleon this looks wrong. Shouldn't this be `meta.name`?

Reply:
This is also the `MIGRATION_PLAN_NAME`, not the `MIGRATION_NAME`. Maybe we should update to reflect the data we're providing.

Reply:
It works correctly. It's also how the hook name has been generated all along:
https://github.com/konveyor/mig-controller/blob/master/pkg/controller/migmigration/hooks.go#L170

```
$ oc get po hooktest-prebackup-8hjzx-znlq7 -o yaml | grep MIGRATION -A1
    - name: MIGRATION_NAMESPACES
      value: registry-images
    - name: MIGRATION_NAME
      value: hooktest

$ oc get migplan
NAME       AGE
hooktest   26h
```

Not objection to changing the env var name.

Reply:
unless I'm doing it wrong and misunderstanding Meta.Name doesn't work
![image](https://user-images.githubusercontent.com/1850500/87470663-16479d80-c5eb-11ea-9f07-3cfa611f1750.png)

Reply:
Ahh, my bad. I forgot realize TypeMeta and ObjectMeta and inlined on the MigPlan struct so they're available from top-level like that. Spec isn't inlined.

```
type MigPlan struct {
	metav1.TypeMeta   `json:",inline"`
	metav1.ObjectMeta `json:"metadata,omitempty"`
```